common/static/common/templates/components/paginated-view.underscore

```diff
@@ -1,4 +1,4 @@
 <div class="sr-is-focusable sr-<%= type %>-view" tabindex="-1"></div>
 <div class="<%= type %>-paging-header"></div>
-<div class="<%= type %>-list"></div>
+<ul class="<%= type %>-list"></ul>
```

Comment:
@frrrances here is the change from div to ul. Note that since thi sis a common component, it will impact more than just Teams.